main.go

```diff
@@ -34,12 +33,10 @@ import (
 	"github.com/oklog/run"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
-	flag "github.com/spf13/pflag"
-	"k8s.io/apimachinery/pkg/util/validation"
-	"k8s.io/kubelet/pkg/apis/deviceplugin/v1beta1"
-
+	"github.com/spf13/viper"
 	"github.com/squat/generic-device-plugin/deviceplugin"
 	"github.com/squat/generic-device-plugin/version"
+	"k8s.io/apimachinery/pkg/util/validation"
```

Comment:
Let's sort these imports and group this party imports above local imports

Reply:
Similarly to the other file, goimports seems to think this is already sorted. Are you using another tool to sort imports?

Reply:
Oh sorry, I should have clarified! The issue was the grouping of the packages. The standard I use in this project, and which we use in the Prometheus community, is:
```go
import (
// Go std lib

// Third party packages

// Packages from this module/repo
)
```

Reply:
Good to know! I've sorted the imports split into 3 sections like that. Look good now?